contrib/scoverage/api/src/mill/contrib/scoverage/api/ScoverageReportWorkerApi.scala

```diff
@@ -28,7 +28,7 @@ trait ScoverageReportWorkerApi {
     ctx.log.error(
       "Binary compatibility stub may cause infinite loops with StackOverflowError. You need to implement: def report(ReportType, Seq[Path], Seq[Path], os.Path): Unit"
     )
-    report(reportType, sources, dataDirs)
+    report(reportType, sources, dataDirs, ctx.workspace)
```

Comment:
This doesn't look right to me. At least in the browser. Your change make the explicit code comment incorrect. If I read this correctly, this report method is meant to be implemented by downstream classes, but to preserve binary compatibility, we can't make it abstract. Hence, we need to call the versions which is guaranteed to be implemented (which is the deprecated one) to avoid any kind of runtime failure.